platform/ui/src/views/StudyList/components/StudyListTable.js

```diff
@@ -275,7 +239,7 @@ const StudyListTable = ({ studies, numOfStudies, filtersMeta }) => {
       <table className="w-full text-white">
```

Comment:
This table are gonna stay as html table or was it missing?

Reply:
Yes, this is gonna be the same as it is for now. As it's a very customized table, with expanded row, different styles, etc... we've decided to only make a reusable table for this one:

![image](https://user-images.githubusercontent.com/1905961/77461429-4c190a00-6de1-11ea-9718-bf9cde52e6ac.png)